OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -163,21 +163,23 @@ std::vector<std::string> LiveFunctionsDataView::GetContextMenu(
     const uint64_t absolute_address =
         capture_data.GetAbsoluteAddress(selected_function, module_map);
 
-    // Is that function actually inside a module of the process (i.e. can we disassemble)?
-    const FunctionInfo* actual_function =
-        capture_data.FindFunctionByAddress(absolute_address, module_map, false);
-    const bool function_exists = actual_function != nullptr;
+    if (GOrbitApp->IsCaptureConnected(capture_data)) {
+      // Is that function actually inside a module of the process (i.e. can we disassemble)?
+      const FunctionInfo* actual_function =
+          capture_data.FindFunctionByAddress(absolute_address, module_map, false);
+      const bool function_exists = actual_function != nullptr;
```

Comment:
I guess, this can now be a check? What happens if you have a process running for long time. Start orbit in a connected state, and load a capture of that process (which is still running). Can you hook then? Currently, we would need to reload the symbols, right?

Reply:
Now: If the same process is still running (determined by isCaptureConnected) then you can select/unselect and disassemble